src/snapshot/prepare_snapshot_received_proposals_airtable.js

```diff
@@ -53,7 +50,6 @@ const prepareProposalsForSnapshot = async (curRound) => {
     proposals.map(async (proposal) => {
       getProposalRecord(proposal, proposals)
       try {
-        const wallet_0x = proposal.get('Wallet Address')
         const proposalStanding = proposal.get('Proposal Standing')
```

Comment:
I am wondering how you went about testing this update.

Reply:
Rolling back this change will cause DAOBot to check all Polygon, BSC and Ethereum every minute and before voting begins. A failure in one of the RPCs - which is common - can cause some problems. @idiom-bytes